.eslintrc

```diff
@@ -25,30 +25,37 @@
       "never",
       { "css": "ignorePackages", "json": "always" }
     ],
-    "react/sort-comp": "off"
+    "react/sort-comp": "off",
+    "import/no-extraneous-dependencies": [
```

Comment:
What's this do?

Reply:
it lets you import a dev dependency within a test